src/pages/user/indext.ts

```diff
@@ -0,0 +1,3 @@
+import Cohorts from './Cohorts';
+
+export default Cohorts;
```

Comment:
Why do you have 2 index files in the same folder
hint this file is `indext.ts` not `index.ts`

and you have an `index.ts` file so you can delete this file and use another one